### DISCUSSION TITLE
Copy & Paste?

### DISCUSSION THREAD

**Discussion by ghost** [erebe/greenclip#134] [Q&A]
Hello,
I'm kinda new to Linux/Arch so this might turn out to be a dumb question.
Does greenclip only save the copied text, and if so it doesn't work for me.
I didn't activate any shortcuts or keys in the rofi config file, because I couldn't find a copy shortcut in the default config. Do I have to activate it first before greenclip saves entries?
Thanks for your help! 
couscous

**Comment by erebe** [comment]
Hello,

Yes you need to start the daemon first. Please consult usage section https://github.com/erebe/greenclip#usage
